### PR TITLE
Cleaning up macros

### PR DESCRIPTION
**Pull request by jasonsandlin**

### CODE REVIEW

**Comment by jicailiu** [comment]
> events::events_service&

here

---
Refers to: Source/Services/Common/xbox_live_context.cpp:174 in 5bc5531. [](commit_id = 5bc5531bbf702a44542415110a719f6ca79da3a9, deletion_comment = False)

**Comment by jicailiu** [comment]
> marketplace::catalog_service&

herehere

---
Refers to: Source/Services/Common/xbox_live_context.cpp:182 in 5bc5531. [](commit_id = 5bc5531bbf702a44542415110a719f6ca79da3a9, deletion_comment = False)

File: Source/Services/Common/xbox_live_context_impl.cpp [hunk 1]
```diff
@@ -205,32 +204,9 @@ m_xboxLiveContextSettings = std::make_shared<xbox::services::xbox_live_context_s
...
-
-        m_signOutContext = m_userContext->user()->GetUserImpl()->add_sign_out_completed_handler(
-                [thisWeakPtr](const system::sign_out_completed_event_args& args)
-        {
-            UNREFERENCED_PARAMETER(args);
-            std::shared_ptr<xbox_live_context_impl> pThis(thisWeakPtr.lock());
-            if (pThis != nullptr)
-            {
-                pThis->real_time_activity_service()->_Close_websocket();
-                presence::presence_writer::get_presence_writer_singleton()->stop_writer(pThis->xbox_live_user_id());
```
Comment (jicailiu) [inline Source/Services/Common/xbox_live_context_impl.cpp]: what about this piece?

Comment (jasonsandlin) [reply Source/Services/Common/xbox_live_context_impl.cpp]: I'm using the XSAPI_NONXDK_CPP_AUTH macro there

Comment (jicailiu) [reply Source/Services/Common/xbox_live_context_impl.cpp]: then it should be XSAPI_NONXDK_CPP_AUTH || XSAPI_NONXDK_WINRT_AUTH?

---
In reply to: [150688820](https://github.com/Microsoft/xbox-live-api/pull/282#discussion_r150688820) [](ancestors = 150688820)

Comment (jicailiu) [reply Source/Services/Common/xbox_live_context_impl.cpp]: or just !XDK

---
In reply to: [150690513](https://github.com/Microsoft/xbox-live-api/pull/282#discussion_r150690513) [](ancestors = 150690513,150688820)

Comment (jasonsandlin) [reply Source/Services/Common/xbox_live_context_impl.cpp]: yeah good point -- XSAPI_NONXDK_CPP_AUTH || XSAPI_NONXDK_WINRT_AUTH

---
In reply to: [150690623](https://github.com/Microsoft/xbox-live-api/pull/282#discussion_r150690623) [](ancestors = 150690623,150690513,150688820)

**Pull request merged**